packages/asc-ui/src/components/Typography/TypographyStyle.ts

```diff
@@ -7,28 +8,31 @@ export type Props = {
   element?: Variant
   fontSize?: number
   styleAs?: keyof Theme.TypographyElements
```

Comment:
Can this be `ElementType` as well?

Reply:
No we want to allow a specific subset of elements here that can only be used in typography,